l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -426,7 +426,7 @@ def _prepareInvoiceLine(self, credit_account_id, line, wt_found=False):
             'account_id': credit_account_id,
             'price_unit': float(line.PrezzoUnitario),
         })
-        if line.Quantita:
+        if line.Quantita is not None:
```

Comment:
cosa ne pensi di un controllo tipo `if float(line.Quantita or 0):`?

Reply:
@GSLabIt `Quantita` non è un campo obbligatorio dell'XML: quando non è presente abbiamo il default a `1`

@sherpya eventualmente potresti aggiungere l'`else` così da scrivere esplicitamente `1` quando `line.Quantita` è `None` e renderlo più chiaro

Reply:
ah capito, grazie

Reply:
@eLBati se non c'è _Quantita_ significa 1? In questo caso _None_ -> 1 va bene, però al solito ste specifiche sono un po' difficile da trovare

Reply:
> se non c'è Quantita significa 1?

Sì. Vedi anche specifiche Assosoftware:

> Se il campo Quantita è omesso, SdI esegue il controllo assumendone il valore pari a 1

Reply:
ho anche eliminato il tag _Quantita_ dalla seconda linea nell'xml di test controllando se la quantità sia 1

Reply:
:+1: